themes/ixTheoTheme/templates/RecordDriver/SolrDefault/result-list.phtml

```diff
@@ -80,11 +80,12 @@
           <? // Handle superior data according to Issue 131 / Rule 5 ?>
           <? $journal_issue = $this->driver->getJournalIssue(); ?>
           <? if (!empty($journal_issue)): ?>
+            <? $superior_ppn_index = !empty($journal_issue[2]) ? 2 : 1; ?>
+            <? $superior_ppn = preg_replace('/\(.*\)/', '', $journal_issue[$superior_ppn_index]); ?>
+            <? unset($journal_issue[$superior_ppn_index]); ?>
```

Comment:
Das Problem war, dass ein Unterfeld in den Marc-Daten fehlen kann. Dadurch gab es keine drei Einträge in $journal_issue. Aber auch ohne dieses Unterfeld kann man eine sinnvolle Verlinkung anzeigen.